indico/modules/events/contributions/templates/display/contribution_display.html

```diff
@@ -29,7 +29,7 @@
             <span itemprop="performers" itemscope itemtype="http://schema.org/Person" class="speaker-item icon-user">
                 {% if user_data.title -%}
                     <span class="speaker-title">
-                        {{ user_data.title }}
+                        {{ user_data.title if user_data.contribution.event.show_title }}
```

Comment:
Why not do this check 2 lines above where we already check if the user has a title?!